tox.ini

```diff
@@ -18,10 +18,10 @@ download = true
 # N.B.: We configure tox to disable its build sdist, then install sdist in venv scheme for all envs
 # with `skip_install = false` (the default). As such, we use a custom `install_command` for all
 # envs that need Pex installed. The install command is tox's default, with the one addition of
-# `{toxinidir}`, which adds `.` to the requirement list handed to Pip to install.
+# `{toxinidir}`, which adds `-e .` to the requirement list handed to Pip to install.
 install_command =
     docs,check,typecheck,py{py27,py35,py36,py37,py38,py39,py310,27,35,36,37,38,39,310,311,312,313}: \
-        python -m pip install {opts} {toxinidir} {packages}
+        python -m pip install {opts} -e {toxinidir} {packages}
```

Comment:
This was needed to ensure non-stale Pex on the test `sys.path`. Toggling back and forth between the aggressive and lazy `import ssl` I noticed the Pex state was sticky. Fallout from #2408 just noticed now.